### PR TITLE
add two lines and four links to melt-with-fork

### PR DESCRIPTION
**Pull request by urock24**
Please review this, @casey-collab

### CODE REVIEW

File: english/fork-knots/melt-with-fork.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 You start tying knots on the fork. Then stick the fork into a marshmallow wall
-and light a free end of a rope up. Yes, the wall slowly melts around the hot fork!
\ No newline at end of file
+and light a free end of a rope up. Yes, the wall slowly melts around the hot fork!
+
+As you are about to step outside you notice an altar in the middle of the room with a chalice on it.
+
+Do you?
+
+[Quickly run outside and explore?](..explore-outside/explore-outside.md)
```
Comment (casey-collab) [inline english/fork-knots/melt-with-fork.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/fork-knots/melt-with-fork.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+[Drink from it?](..fork-jam/choco-chalice/choco-chalice.md)
```
Comment (casey-collab) [inline english/fork-knots/melt-with-fork.md]: This link is also invalid.

File: english/fork-knots/melt-with-fork.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+
+[Smash the altar](..fork-jam/altar-smash/altar-smash.md)
```
Comment (casey-collab) [inline english/fork-knots/melt-with-fork.md]: This link is also invalid.

File: english/fork-knots/melt-with-fork.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+
+[Pray in the altar?](..fork-jam/altar-pray/altar-pray.md)
```
Comment (casey-collab) [inline english/fork-knots/melt-with-fork.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by urock24** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :wind_chime:

**Pull request merged**